testing-and-tdd/jasmine-testing.md

```diff
@@ -25,173 +25,362 @@ Jasmine is a Behavior Driven Development testing framework for JavaScript. It do
 
 *Participants will be able to:*
 
-- create a testing structure
-- create assertion functions
+- Create a testing structure with Jasmine
+- Create assertion functions
 - Generate, display and watch tests
 
 ### Specific Things to Learn
 
 - [Jasmine](https://jasmine.github.io/index.html)
   - install Jasmine
-  - create tests using `toBe`, `beforeEach`, `afterEach`
-- [Mocha](https://mochajs.org/)
-  - install Mocha
-  - create tests using assert
-- [Chai](https://www.chaijs.com/)
-  - install Chai
-  - require library for assertions
+  - create tests using `expect` & matchers like `toBe`, `toContain`, and `toBeDefined`
 
 ### Materials
 
-- [Jasmine Testing Slideshow](https://docs.google.com/presentation/d/1V9WEprK9j61V0S1lYtcJOXNTDMpziVMyUKHuMhpP1W0/edit?usp=sharing)
+- [Jasmine and Mocha/Chai Testing Slideshow](https://docs.google.com/presentation/d/1jMkVbUkZS40z7kAoC1mPIt8RwfF8-GRL8oeee25UzYw)
+    - Note: you can skip over the Mocha / Chai sections of the slide show for now. We'll go over that in the next lesson!
 
 ### Lesson
 
 #### Jasmine Test
-Jasmine is a behavior-driven development (BDD) for testing JavaScript code. Jasmine has no external dependencies and does not require a DOM. 
+Jasmine is a behavior-driven development (BDD) framework for testing JavaScript code. (Note: BDD is a specific style of test-driven development that tests the behavior of the code from the user's perspective.) Jasmine has no external dependencies and does not require a DOM, which means that it's good for getting tests up and running quickly.
 
-You should write the test first then construct code to pass the test. If developers don't start with test it can be more time consuming to write them later.
+As we learned in the last lesson on test-driven development (TDD), one way to ensure that your code is well-tested is to start by writing a test for the behavior you want, watch it fail, and finally write the code to make it pass (the Red-Green-Refactor pattern). Though working in a TDD style may feel slower at first, it can save you time in the long run by ensuring that your code won't break. We'll be working in a TDD style through this lesson.
 
-*install jasmine*
-- `npm install --global jasmine` will install jasmine globally.
-- `npm install --save-dev jasmine` for testing in your future projects. This saves Jasmine in the project you are using so that it can be shared with others when they clone the repository.
+Let's get started by setting up a new project with Jasmine tests.
 
-*initialize project*
-- `jasmine init` will work if jasmine was installed globably.
-- `node node_modules/jasmine/bin/jasmine init` will work if jasmine was installed locally.
+*Create a new project*
+- `mkdir jasmine-practice` - create a folder for your new project
```

Comment:
i'm glad it doesn't say 'globably' anymore 😆

Reply:
😆